src/smc-webapp/course/actions.coffee

```diff
@@ -314,7 +314,7 @@ exports.CourseActions = class CourseActions extends Actions
 
     # Takes an item_name and the id of the time
     # item_name should be one of
-    # ['student', 'assignment', 'peer_config', handout']
+    # ['student', 'assignment', 'peer_config', handout', 'skip_grading']
```

Comment:
self reviewing this, ha, well, this 'skip_grading' shouldn't be here in the comment. leftover from early attempts.